tests/rptest/tests/cloud_storage_scrubber_test.py

```diff
@@ -637,6 +678,10 @@ def test_scrubber(self, cloud_storage_type):
         self._produce()
         self._assert_no_anomalies()
 
+        # Add a randomly generated inv. report to the mix. This will have some keys missing, but scrubbing
```

Comment:
nit: it doesn't seem to be randomly generated?

Reply:
Yeah, the randomness didn't work with the test. I will update the comment.